go.mod

```diff
@@ -30,3 +30,5 @@ require (
 	google.golang.org/appengine v1.6.7 // indirect
 	google.golang.org/genproto v0.0.0-20220505152158-f39f71e6c8f3 // indirect
 )
+
+replace gopkg.in/yaml.v3 v3.0.0-20200313102051-9f266ea9e77c => gopkg.in/yaml.v3 v3.0.0
```

Comment:
Could we upgrade to [3.0.1](https://github.com/go-yaml/yaml/releases/tag/v3.0.1)?

Reply:
I don't see why not, but to fix the vulnerability, 3.0.0 is enough.